HelperMethods.rb

```diff
@@ -79,14 +85,16 @@ def register_user
 
     @driver.find_element(:class, 'register').click
     @wait.until {@driver.find_element(:id, 'new_user').displayed? }
-
-    @login = ('login' + rand(99999).to_s)
+    #@login = ('login' + rand(99999).to_s)
+    user = Users.new
```

Comment:
I would advise you to find a good place for creating user and placing it into @user variable to make it accessible everywhere in your tests